### PR TITLE
sina: add 1.11 version

### PR DESCRIPTION
**Pull request by HaluskaR**

### CODE REVIEW

**Review by estebanpauli — DISMISSED** [review]

**Comment by murray55** [comment]
Changes look good! Although I can't approve for some reason.

File: var/spack/repos/builtin/packages/sina/package.py [hunk 1]
```diff
@@ -21,6 +21,7 @@ class Sina(CachedCMakePackage):
     ]
 
     version('1.10.0', sha256='b34379ce8cc5eca5a0f16893053fac75be14c2109d1beed4c6d48e11f9b281c7')
+    version('1.11.0', sha256='a080f6583101696a6354940f00646ef892e1b2f8fc3f1b1907ba1c3ade6d4b47')
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/sina/package.py:23]: The Spack convention is for newer versions to appear before older ones.  Please move this *above* `1.10.0`.

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/sina/package.py:23]: Confirmed the sha256.

**Review by tldahlgren — CHANGES_REQUESTED** [review]

**Review by tldahlgren — DISMISSED** [review]

**Comment by haampie** [comment]
@spackbot run pipeline

**Comment by tldahlgren** [comment]
Please rebase to develop so you can pick up the pipeline fix.

**Comment by tldahlgren** [comment]
Closing and re-opening to force required audit checks to be re-run.

**Review by tldahlgren — APPROVED** [review]
LGTM

**Pull request merged**